bitsail-components/bitsail-component-metrics/bitsail-component-metrics-prometheus/src/main/java/com/bytedance/bitsail/component/metrics/prometheus/option/PrometheusOptions.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 Bytedance Ltd. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bytedance.bitsail.component.metrics.prometheus.option;
+
+import com.bytedance.bitsail.common.option.CommonOptions;
+import com.bytedance.bitsail.common.option.ConfigOption;
+import com.bytedance.bitsail.common.option.ConfigOptions;
+import com.bytedance.bitsail.common.option.WriterOptions;
+
+public interface PrometheusOptions extends WriterOptions.BaseWriterOptions {
```

Comment:
should we put metric related option into common?

Reply:
It will be many different metric related option in different metric gather, it will below common scope in job config like that:
![image](https://user-images.githubusercontent.com/47937942/217173128-7b450aaf-17b8-485b-bebf-810cdb5ea2cd.png)
but I use PrometheusOptions in order to make the option file more clearly.

Reply:
why is this still extend the writer option